tests/test_datasf.py

```diff
@@ -0,0 +1,16 @@
+"""Tests for Records resource"""
+from service.models.datasf import DataSF
+
+def test_datasf_filter():
+    """ Test DataSF filter """
+    datasf = DataSF()
+    response = datasf.filter('https://data.sfgov.org/resource/wwmu-gmzc.json')
```

Comment:
more of an integration test than a unit test, right?

Reply:
that's correct. created a unit test for this. 6291eb385e4a0597a18cbc47e36f00e6b7b5e290